### ISSUE TITLE
vendor: warning in go-libedit

### ISSUE THREAD

**Issue by tschottdorf** [cockroachdb/cockroach#20867]
Seen during `./build/builder.sh make build` on `darwin`.

```
vendor/github.com/knz/go-libedit/unix/src/c-libedit/search.c:607:20: warning: passing 'wint_t *' (aka 'unsigned int *') to parameter of type 'wchar_t *' (aka 'int *') converts between pointers to integer types with different sign [-Wpointer-sign]
vendor/github.com/knz/go-libedit/unix/src/c-libedit/histedit.h:267:37: note: passing argument to parameter here
```

@knz said:

> interesting. I checked the code and it's a bug in the upstream code. I can make a patch, but apparently it's innocuous.

PS: I'm filing this here instead of `go-libedit` so that others who run into it find it.

**Comment by knz** [comment]
Tentatively labeling 2.0, might slip into 2.1.

**Comment by benesch** [comment]
Fixed by knz/go-libedit#5. Didn’t make 2.0, but will make 2.1.
